packages/documentation/pages/usage/components/standard-table.vue

```diff
@@ -169,13 +174,84 @@ export default defineComponent({
 				isSortable: true,
 				label: 'Todo',
 			},
+			{
+				display: { type: 'text' },
+				filterFn: (row, columnId, filterValue) =>
+					row.getValue(columnId) === filterValue,
+				getData: (row: TodoRow) => row.priority,
+				id: 'priority',
+				isSortable: true,
+				label: 'Priority',
+			},
+			{
+				display: { decimalPlaces: 2, type: 'numerical' },
+				filterFn: (row, columnId, filterValue) => {
```

Comment:
`filterFn` is exposed in the `Column` type. Right now its implementation is on the user side. Could be moved into the standard table hook at the expense of leaving less flexibility in terms of the implementation/logic for the filters/values in the table.